packages/cashier-v2/src/constants/constants.ts

```diff
@@ -0,0 +1,88 @@
+import { TMT5LandingCompanyName } from '../hooks/types';
+
+type TDefinedMT5LandingCompanyName = Exclude<TMT5LandingCompanyName, 'malta' | 'seychelles' | undefined>;
+
+interface TDefinedMT5LandingCompanyDetails {
+    name: TDefinedMT5LandingCompanyName;
+    shortcode: string;
+}
+
+export const LandingCompanyDetails: Record<TDefinedMT5LandingCompanyName, TDefinedMT5LandingCompanyDetails> = {
+    bvi: {
```

Comment:
Are we in the process of getting this into an API endpoint? With the yaml config's too?

Reply:
Hi @markwylde-deriv,
I am not sure about this. This is a simple mapper which helps us to show the correct market type besides the name of the accounts in the cashier transfer dropdowns.
<img width="973" alt="Screenshot 2024-03-14 at 5 48 57 PM" src="https://github.com/binary-com/deriv-app/assets/125039206/cef3a1c1-8126-4e4d-8867-40fb3568e571">

Reply:
PS: We do have an API call for getting the landing company details but it does not provide the required name to be shown along with the account names.

Reply:
Yeah I get it. We just need to keep pushing to not hard code this stuff into the deriv-app. It all needs to come from config or the API as they can change at any time.